### ISSUE TITLE
Does Learn contain a complete Sandbox for learning?

### ISSUE THREAD

**Issue by gman8321** [MicrosoftDocs/TeamBlogFeedback#127]
I use SalesForce Trailhead training, which gives a truly complete sandbox environment. Everything that SalesForce offers is in the Sandbox. It doesn't appear to be the case with Microsoft Learn. I tried with the Complete an approval flow training.  First part you have to create a SharePoint list. Where is the SharePoint list in the sandbox? without one, the training module is useless. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 26c7d914-e786-d705-977d-4f5fe050f420
* Version Independent ID: 90e0d126-c3ff-00ec-b910-528c64a8c9db
* Content: [Introducing Microsoft.com/Learn](https://docs.microsoft.com/en-us/teamblog/introducing-ms-learn#we-want-your-feedback)
* Content Source: [teamblog/introducing-ms-learn.md](https://github.com/MicrosoftDocs/blog/blob/live/teamblog/introducing-ms-learn.md)
* Product: **team-blog**
* GitHub Login: @jeffsand
* Microsoft Alias: **jeffsand**

**Comment by f1point2** [comment]
No; there isn't a fully fledged sandbox. Just the various Azure credits available; most notably through Visual Studio Subscriptions (which don't come cheap and certainly don't count as free).

When you work through learn; some of the units have a sandbox provided to allow individuals to work through specific tasks. E.g.
https://docs.microsoft.com/en-us/learn/modules/welcome-to-azure/3-create-a-vm?pivots=windows-cloud

This are very basic; hopefully we will see more challenging and open ended scenarios as the platform matures.

**Comment by egallagh** [comment]
And there are various free services and free trial offerings available. There are also benefits that may be available through an employer, educational institution or Microsoft Partnership.

**Comment by StevieNookz** [comment]
Hey, will Microsoft Learn be merged with or replace Virtual Academy? https://mva.microsoft.com/

**Comment by mlottner** [comment]
@gman8321 Thanks for taking the time to share your blog feedback. 
I've reassigned this issue to the content team from Learn. 
In the future, make sure to submit your page or platform feedback on the specific tool or product to ensure you get the most timely response possible. 
Appreciate your contribution to our community!
